### ISSUE TITLE
使用slim编写，loss变成了890347，吓人

### ISSUE THREAD

**Issue by aoguren** [luyishisi/tensorflow#6]
with slim.arg_scope([slim.conv2d, slim.fully_connected],
                        activation_fn=tf.nn.relu,
                        biases_initializer=tf.random_normal_initializer,
                        weights_initializer=tf.random_normal_initializer,
                        ):

        conv1 = slim.conv2d(x, 32, [3, 3], 1)
        pool1 = slim.max_pool2d(conv1, [2, 2], 2, padding='SAME')
        drop1 = slim.dropout(pool1, keep_prob=keep_prob)
        conv2 = slim.conv2d(drop1, 64, [3, 3], 1)
        pool2 = slim.max_pool2d(conv2, [2, 2], 2, padding='SAME')
        drop2 = slim.dropout(pool2, keep_prob=keep_prob)
        conv3 = slim.conv2d(drop2, 64, [3, 3], 1)
        pool3 = slim.max_pool2d(conv3, [2, 2], 2, padding='SAME')
        drop3 = slim.dropout(pool3, keep_prob=keep_prob)
        flatten = slim.flatten(drop3)
        dense1 = slim.fully_connected(flatten, 1024)
        drop4 = slim.dropout(dense1, keep_prob=keep_prob)
        out = slim.fully_connected(drop4, MAX_CAPTCHA*CHAR_SET_LEN, activation_fn=None)
    return out
贴上代码，其它都一样，但是训练时初始的loss超级大，@luyishisi能帮忙分析一下吗？
